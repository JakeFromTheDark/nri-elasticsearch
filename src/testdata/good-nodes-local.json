{
	"_nodes": {
		"total": 1,
		"successful": 1,
		"failed": 0
	},
	"cluster_name": "es6-dev-cluster",
	"nodes": {
		"z9ZPp87vT92qG1cRVRIcMQ": {
			"name": "es6c-dev-m1.bluemedora.localnet",
			"transport_address": "10.77.10.83:9300",
			"host": "10.77.10.83",
			"ip": "10.77.10.83",
			"version": "6.3.2",
			"build_flavor": "default",
			"build_type": "rpm",
			"build_hash": "053779d",
			"total_indexing_buffer": 105630924,
			"roles": [
				"master"
			],
			"attributes": {
				"xpack.installed": "true"
			},
			"settings": {
				"pidfile": "/var/run/elasticsearch/elasticsearch.pid",
				"cluster": {
					"name": "es6-dev-cluster"
				},
				"node": {
					"name": "es6c-dev-m1.bluemedora.localnet",
					"attr": {
						"xpack": {
							"installed": "true"
						}
					},
					"data": "false",
					"ingest": "false",
					"master": "true",
					"ml": "false"
				},
				"path": {
					"data": [
						"/home/elasticsearch/data"
					],
					"logs": "/home/elasticsearch/logs",
					"home": "/usr/share/elasticsearch"
				},
				"discovery": {
					"zen": {
						"minimum_master_nodes": "2",
						"ping": {
							"unicast": {
								"hosts": [
									"10.77.10.82",
									"10.77.10.83",
									"10.77.10.84"
								]
							}
						}
					}
				},
				"action": {
					"auto_create_index": ".security,.monitoring*,.watches,.triggered_watches,.watcher-history*,.ml*"
				},
				"client": {
					"type": "node"
				},
				"http": {
					"type": "security4",
					"type.default": "netty4"
				},
				"transport": {
					"type": "security4",
					"features": {
						"x-pack": "true"
					},
					"type.default": "netty4"
				},
				"xpack": {
					"ml": {
						"enabled": "false"
					}
				},
				"network": {
					"host": "_site_"
				}
			},
			"os": {
				"refresh_interval_in_millis": 1000,
				"name": "Linux",
				"arch": "amd64",
				"version": "3.10.0-862.9.1.el7.x86_64",
				"available_processors": 2,
				"allocated_processors": 2
			},
			"process": {
				"refresh_interval_in_millis": 1000,
				"id": 15171,
				"mlockall": false
			},
			"jvm": {
				"pid": 15171,
				"version": "1.8.0_171",
				"vm_name": "OpenJDK 64-Bit Server VM",
				"vm_version": "25.171-b10",
				"vm_vendor": "Oracle Corporation",
				"start_time_in_millis": 1533068098515,
				"mem": {
					"heap_init_in_bytes": 1073741824,
					"heap_max_in_bytes": 1056309248,
					"non_heap_init_in_bytes": 2555904,
					"non_heap_max_in_bytes": 0,
					"direct_max_in_bytes": 1056309248
				},
				"gc_collectors": [
					"ParNew",
					"ConcurrentMarkSweep"
				],
				"memory_pools": [
					"Code Cache",
					"Metaspace",
					"Compressed Class Space",
					"Par Eden Space",
					"Par Survivor Space",
					"CMS Old Gen"
				],
				"using_compressed_ordinary_object_pointers": "true",
				"input_arguments": [
					"-Xms1g",
					"-Xmx1g",
					"-Dcom.sun.management.jmxremote",
					"-Dcom.sun.management.jmxremote.port=9999",
					"-Dcom.sun.management.jmxremote.rmi.port=9998",
					"-Dcom.sun.management.jmxremote.local.only=false",
					"-Dcom.sun.management.jmxremote.authenticate=false",
					"-Dcom.sun.management.jmxremote.ssl=false",
					"-XX:+UseConcMarkSweepGC",
					"-XX:CMSInitiatingOccupancyFraction=75",
					"-XX:+UseCMSInitiatingOccupancyOnly",
					"-XX:+AlwaysPreTouch",
					"-Xss1m",
					"-Djava.awt.headless=true",
					"-Dfile.encoding=UTF-8",
					"-Djna.nosys=true",
					"-XX:-OmitStackTraceInFastThrow",
					"-Dio.netty.noUnsafe=true",
					"-Dio.netty.noKeySetOptimization=true",
					"-Dio.netty.recycler.maxCapacityPerThread=0",
					"-Dlog4j.shutdownHookEnabled=false",
					"-Dlog4j2.disable.jmx=true",
					"-Djava.io.tmpdir=/tmp/elasticsearch.ylBnmudC",
					"-XX:+HeapDumpOnOutOfMemoryError",
					"-XX:HeapDumpPath=/var/lib/elasticsearch",
					"-XX:ErrorFile=/var/log/elasticsearch/hs_err_pid%p.log",
					"-XX:+PrintGCDetails",
					"-XX:+PrintGCDateStamps",
					"-XX:+PrintTenuringDistribution",
					"-XX:+PrintGCApplicationStoppedTime",
					"-Xloggc:/var/log/elasticsearch/gc.log",
					"-XX:+UseGCLogFileRotation",
					"-XX:NumberOfGCLogFiles=32",
					"-XX:GCLogFileSize=64m",
					"-Des.path.home=/usr/share/elasticsearch",
					"-Des.path.conf=/etc/elasticsearch",
					"-Des.distribution.flavor=default",
					"-Des.distribution.type=rpm"
				]
			},
			"thread_pool": {
				"watcher": {
					"type": "fixed",
					"min": 1,
					"max": 1,
					"queue_size": 1000
				},
				"force_merge": {
					"type": "fixed",
					"min": 1,
					"max": 1,
					"queue_size": -1
				},
				"security-token-key": {
					"type": "fixed",
					"min": 1,
					"max": 1,
					"queue_size": 1000
				},
				"fetch_shard_started": {
					"type": "scaling",
					"min": 1,
					"max": 4,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"listener": {
					"type": "fixed",
					"min": 1,
					"max": 1,
					"queue_size": -1
				},
				"index": {
					"type": "fixed",
					"min": 2,
					"max": 2,
					"queue_size": 200
				},
				"refresh": {
					"type": "scaling",
					"min": 1,
					"max": 1,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"generic": {
					"type": "scaling",
					"min": 4,
					"max": 128,
					"keep_alive": "30s",
					"queue_size": -1
				},
				"rollup_indexing": {
					"type": "fixed",
					"min": 4,
					"max": 4,
					"queue_size": 4
				},
				"warmer": {
					"type": "scaling",
					"min": 1,
					"max": 1,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"search": {
					"type": "fixed_auto_queue_size",
					"min": 4,
					"max": 4,
					"queue_size": 1000
				},
				"flush": {
					"type": "scaling",
					"min": 1,
					"max": 1,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"fetch_shard_store": {
					"type": "scaling",
					"min": 1,
					"max": 4,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"management": {
					"type": "scaling",
					"min": 1,
					"max": 5,
					"keep_alive": "5m",
					"queue_size": -1
				},
				"get": {
					"type": "fixed",
					"min": 2,
					"max": 2,
					"queue_size": 1000
				},
				"analyze": {
					"type": "fixed",
					"min": 1,
					"max": 1,
					"queue_size": 16
				},
				"write": {
					"type": "fixed",
					"min": 2,
					"max": 2,
					"queue_size": 200
				},
				"snapshot": {
					"type": "scaling",
					"min": 1,
					"max": 1,
					"keep_alive": "5m",
					"queue_size": -1
				}
			},
			"transport": {
				"bound_address": [
					"10.77.10.83:9300"
				],
				"publish_address": "10.77.10.83:9300",
				"profiles": {}
			},
			"http": {
				"bound_address": [
					"10.77.10.83:9200"
				],
				"publish_address": "10.77.10.83:9200",
				"max_content_length_in_bytes": 104857600
			},
			"plugins": [
				{
					"name": "analysis-icu",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU relates analysis components.",
					"classname": "org.elasticsearch.plugin.analysis.icu.AnalysisICUPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				}
			],
			"modules": [
				{
					"name": "aggs-matrix-stats",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Adds aggregations whose input are a list of numeric fields and output includes a matrix.",
					"classname": "org.elasticsearch.search.aggregations.matrix.MatrixAggregationPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "analysis-common",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Adds \"built in\" analyzers to Elasticsearch.",
					"classname": "org.elasticsearch.analysis.common.CommonAnalysisPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "ingest-common",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Module for ingest processors that do not require additional security permissions or have large dependencies and resources",
					"classname": "org.elasticsearch.ingest.common.IngestCommonPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "lang-expression",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Lucene expressions integration for Elasticsearch",
					"classname": "org.elasticsearch.script.expression.ExpressionPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "lang-mustache",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Mustache scripting integration for Elasticsearch",
					"classname": "org.elasticsearch.script.mustache.MustachePlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "lang-painless",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "An easy, safe and fast scripting language for Elasticsearch",
					"classname": "org.elasticsearch.painless.PainlessPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "mapper-extras",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Adds advanced field mappers",
					"classname": "org.elasticsearch.index.mapper.MapperExtrasPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "parent-join",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "This module adds the support parent-child queries and aggregations",
					"classname": "org.elasticsearch.join.ParentJoinPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "percolator",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Percolator module adds capability to index queries and query these queries by specifying documents",
					"classname": "org.elasticsearch.percolator.PercolatorPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "rank-eval",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "The Rank Eval module adds APIs to evaluate ranking quality.",
					"classname": "org.elasticsearch.index.rankeval.RankEvalPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "reindex",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "The Reindex module adds APIs to reindex from one index to another or update documents in place.",
					"classname": "org.elasticsearch.index.reindex.ReindexPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "repository-url",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Module for URL repository",
					"classname": "org.elasticsearch.plugin.repository.url.URLRepositoryPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "transport-netty4",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Netty 4 based transport implementation",
					"classname": "org.elasticsearch.transport.Netty4Plugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "tribe",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Tribe module",
					"classname": "org.elasticsearch.tribe.TribePlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "x-pack-core",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Core",
					"classname": "org.elasticsearch.xpack.core.XPackPlugin",
					"extended_plugins": [],
					"has_native_controller": false
				},
				{
					"name": "x-pack-deprecation",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Deprecation",
					"classname": "org.elasticsearch.xpack.deprecation.Deprecation",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-graph",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Graph",
					"classname": "org.elasticsearch.xpack.graph.Graph",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-logstash",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Logstash",
					"classname": "org.elasticsearch.xpack.logstash.Logstash",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-ml",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Machine Learning",
					"classname": "org.elasticsearch.xpack.ml.MachineLearning",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": true
				},
				{
					"name": "x-pack-monitoring",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Monitoring",
					"classname": "org.elasticsearch.xpack.monitoring.Monitoring",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-rollup",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Rollup",
					"classname": "org.elasticsearch.xpack.rollup.Rollup",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-security",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Security",
					"classname": "org.elasticsearch.xpack.security.Security",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-sql",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "The Elasticsearch plugin that powers SQL for Elasticsearch",
					"classname": "org.elasticsearch.xpack.sql.plugin.SqlPlugin",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-upgrade",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Upgrade",
					"classname": "org.elasticsearch.xpack.upgrade.Upgrade",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				},
				{
					"name": "x-pack-watcher",
					"version": "6.3.2",
					"elasticsearch_version": "6.3.2",
					"java_version": "1.8",
					"description": "Elasticsearch Expanded Pack Plugin - Watcher",
					"classname": "org.elasticsearch.xpack.watcher.Watcher",
					"extended_plugins": [
						"x-pack-core"
					],
					"has_native_controller": false
				}
			],
			"ingest": {
				"processors": [
					{
						"type": "append"
					},
					{
						"type": "convert"
					},
					{
						"type": "date"
					},
					{
						"type": "date_index_name"
					},
					{
						"type": "dot_expander"
					},
					{
						"type": "fail"
					},
					{
						"type": "foreach"
					},
					{
						"type": "grok"
					},
					{
						"type": "gsub"
					},
					{
						"type": "join"
					},
					{
						"type": "json"
					},
					{
						"type": "kv"
					},
					{
						"type": "lowercase"
					},
					{
						"type": "remove"
					},
					{
						"type": "rename"
					},
					{
						"type": "script"
					},
					{
						"type": "set"
					},
					{
						"type": "set_security_user"
					},
					{
						"type": "sort"
					},
					{
						"type": "split"
					},
					{
						"type": "trim"
					},
					{
						"type": "uppercase"
					},
					{
						"type": "urldecode"
					}
				]
			}
		}
	}
}