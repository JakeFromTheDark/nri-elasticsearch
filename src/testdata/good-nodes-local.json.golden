{"attributes":{"xpack.installed":"true"},"build_flavor":"default","build_hash":"053779d","build_type":"rpm","host":"10.77.10.83","http":{"bound_address":["10.77.10.83:9200"],"max_content_length_in_bytes":104857600,"publish_address":"10.77.10.83:9200"},"ingest":{"processors":[{"type":"append"},{"type":"convert"},{"type":"date"},{"type":"date_index_name"},{"type":"dot_expander"},{"type":"fail"},{"type":"foreach"},{"type":"grok"},{"type":"gsub"},{"type":"join"},{"type":"json"},{"type":"kv"},{"type":"lowercase"},{"type":"remove"},{"type":"rename"},{"type":"script"},{"type":"set"},{"type":"set_security_user"},{"type":"sort"},{"type":"split"},{"type":"trim"},{"type":"uppercase"},{"type":"urldecode"}]},"ip":"10.77.10.83","jvm":{"gc_collectors":["ParNew","ConcurrentMarkSweep"],"input_arguments":["-Xms1g","-Xmx1g","-Dcom.sun.management.jmxremote","-Dcom.sun.management.jmxremote.port=9999","-Dcom.sun.management.jmxremote.rmi.port=9998","-Dcom.sun.management.jmxremote.local.only=false","-Dcom.sun.management.jmxremote.authenticate=false","-Dcom.sun.management.jmxremote.ssl=false","-XX:+UseConcMarkSweepGC","-XX:CMSInitiatingOccupancyFraction=75","-XX:+UseCMSInitiatingOccupancyOnly","-XX:+AlwaysPreTouch","-Xss1m","-Djava.awt.headless=true","-Dfile.encoding=UTF-8","-Djna.nosys=true","-XX:-OmitStackTraceInFastThrow","-Dio.netty.noUnsafe=true","-Dio.netty.noKeySetOptimization=true","-Dio.netty.recycler.maxCapacityPerThread=0","-Dlog4j.shutdownHookEnabled=false","-Dlog4j2.disable.jmx=true","-Djava.io.tmpdir=/tmp/elasticsearch.ylBnmudC","-XX:+HeapDumpOnOutOfMemoryError","-XX:HeapDumpPath=/var/lib/elasticsearch","-XX:ErrorFile=/var/log/elasticsearch/hs_err_pid%p.log","-XX:+PrintGCDetails","-XX:+PrintGCDateStamps","-XX:+PrintTenuringDistribution","-XX:+PrintGCApplicationStoppedTime","-Xloggc:/var/log/elasticsearch/gc.log","-XX:+UseGCLogFileRotation","-XX:NumberOfGCLogFiles=32","-XX:GCLogFileSize=64m","-Des.path.home=/usr/share/elasticsearch","-Des.path.conf=/etc/elasticsearch","-Des.distribution.flavor=default","-Des.distribution.type=rpm"],"mem":{"direct_max_in_bytes":1056309248,"heap_init_in_bytes":1073741824,"heap_max_in_bytes":1056309248,"non_heap_init_in_bytes":2555904,"non_heap_max_in_bytes":0},"memory_pools":["Code Cache","Metaspace","Compressed Class Space","Par Eden Space","Par Survivor Space","CMS Old Gen"],"pid":15171,"start_time_in_millis":1533068098515,"using_compressed_ordinary_object_pointers":"true","version":"1.8.0_171","vm_name":"OpenJDK 64-Bit Server VM","vm_vendor":"Oracle Corporation","vm_version":"25.171-b10"},"modules":[{"classname":"org.elasticsearch.search.aggregations.matrix.MatrixAggregationPlugin","description":"Adds aggregations whose input are a list of numeric fields and output includes a matrix.","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"aggs-matrix-stats","version":"6.3.2"},{"classname":"org.elasticsearch.analysis.common.CommonAnalysisPlugin","description":"Adds \"built in\" analyzers to Elasticsearch.","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"analysis-common","version":"6.3.2"},{"classname":"org.elasticsearch.ingest.common.IngestCommonPlugin","description":"Module for ingest processors that do not require additional security permissions or have large dependencies and resources","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"ingest-common","version":"6.3.2"},{"classname":"org.elasticsearch.script.expression.ExpressionPlugin","description":"Lucene expressions integration for Elasticsearch","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"lang-expression","version":"6.3.2"},{"classname":"org.elasticsearch.script.mustache.MustachePlugin","description":"Mustache scripting integration for Elasticsearch","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"lang-mustache","version":"6.3.2"},{"classname":"org.elasticsearch.painless.PainlessPlugin","description":"An easy, safe and fast scripting language for Elasticsearch","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"lang-painless","version":"6.3.2"},{"classname":"org.elasticsearch.index.mapper.MapperExtrasPlugin","description":"Adds advanced field mappers","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"mapper-extras","version":"6.3.2"},{"classname":"org.elasticsearch.join.ParentJoinPlugin","description":"This module adds the support parent-child queries and aggregations","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"parent-join","version":"6.3.2"},{"classname":"org.elasticsearch.percolator.PercolatorPlugin","description":"Percolator module adds capability to index queries and query these queries by specifying documents","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"percolator","version":"6.3.2"},{"classname":"org.elasticsearch.index.rankeval.RankEvalPlugin","description":"The Rank Eval module adds APIs to evaluate ranking quality.","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"rank-eval","version":"6.3.2"},{"classname":"org.elasticsearch.index.reindex.ReindexPlugin","description":"The Reindex module adds APIs to reindex from one index to another or update documents in place.","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"reindex","version":"6.3.2"},{"classname":"org.elasticsearch.plugin.repository.url.URLRepositoryPlugin","description":"Module for URL repository","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"repository-url","version":"6.3.2"},{"classname":"org.elasticsearch.transport.Netty4Plugin","description":"Netty 4 based transport implementation","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"transport-netty4","version":"6.3.2"},{"classname":"org.elasticsearch.tribe.TribePlugin","description":"Tribe module","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"tribe","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.core.XPackPlugin","description":"Elasticsearch Expanded Pack Plugin - Core","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"x-pack-core","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.deprecation.Deprecation","description":"Elasticsearch Expanded Pack Plugin - Deprecation","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-deprecation","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.graph.Graph","description":"Elasticsearch Expanded Pack Plugin - Graph","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-graph","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.logstash.Logstash","description":"Elasticsearch Expanded Pack Plugin - Logstash","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-logstash","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.ml.MachineLearning","description":"Elasticsearch Expanded Pack Plugin - Machine Learning","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":true,"java_version":"1.8","name":"x-pack-ml","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.monitoring.Monitoring","description":"Elasticsearch Expanded Pack Plugin - Monitoring","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-monitoring","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.rollup.Rollup","description":"Elasticsearch Expanded Pack Plugin - Rollup","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-rollup","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.security.Security","description":"Elasticsearch Expanded Pack Plugin - Security","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-security","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.sql.plugin.SqlPlugin","description":"The Elasticsearch plugin that powers SQL for Elasticsearch","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-sql","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.upgrade.Upgrade","description":"Elasticsearch Expanded Pack Plugin - Upgrade","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-upgrade","version":"6.3.2"},{"classname":"org.elasticsearch.xpack.watcher.Watcher","description":"Elasticsearch Expanded Pack Plugin - Watcher","elasticsearch_version":"6.3.2","extended_plugins":["x-pack-core"],"has_native_controller":false,"java_version":"1.8","name":"x-pack-watcher","version":"6.3.2"}],"name":"es6c-dev-m1.bluemedora.localnet","os":{"allocated_processors":2,"arch":"amd64","available_processors":2,"name":"Linux","refresh_interval_in_millis":1000,"version":"3.10.0-862.9.1.el7.x86_64"},"plugins":[{"classname":"org.elasticsearch.plugin.analysis.icu.AnalysisICUPlugin","description":"The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU relates analysis components.","elasticsearch_version":"6.3.2","extended_plugins":[],"has_native_controller":false,"java_version":"1.8","name":"analysis-icu","version":"6.3.2"}],"process":{"id":15171,"mlockall":false,"refresh_interval_in_millis":1000},"roles":["master"],"settings":{"action":{"auto_create_index":".security,.monitoring*,.watches,.triggered_watches,.watcher-history*,.ml*"},"client":{"type":"node"},"cluster":{"name":"es6-dev-cluster"},"discovery":{"zen":{"minimum_master_nodes":"2","ping":{"unicast":{"hosts":["10.77.10.82","10.77.10.83","10.77.10.84"]}}}},"http":{"type":"security4","type.default":"netty4"},"network":{"host":"_site_"},"node":{"attr":{"xpack":{"installed":"true"}},"data":"false","ingest":"false","master":"true","ml":"false","name":"es6c-dev-m1.bluemedora.localnet"},"path":{"data":["/home/elasticsearch/data"],"home":"/usr/share/elasticsearch","logs":"/home/elasticsearch/logs"},"pidfile":"/var/run/elasticsearch/elasticsearch.pid","transport":{"features":{"x-pack":"true"},"type":"security4","type.default":"netty4"},"xpack":{"ml":{"enabled":"false"}}},"thread_pool":{"analyze":{"max":1,"min":1,"queue_size":16,"type":"fixed"},"fetch_shard_started":{"keep_alive":"5m","max":4,"min":1,"queue_size":-1,"type":"scaling"},"fetch_shard_store":{"keep_alive":"5m","max":4,"min":1,"queue_size":-1,"type":"scaling"},"flush":{"keep_alive":"5m","max":1,"min":1,"queue_size":-1,"type":"scaling"},"force_merge":{"max":1,"min":1,"queue_size":-1,"type":"fixed"},"generic":{"keep_alive":"30s","max":128,"min":4,"queue_size":-1,"type":"scaling"},"get":{"max":2,"min":2,"queue_size":1000,"type":"fixed"},"index":{"max":2,"min":2,"queue_size":200,"type":"fixed"},"listener":{"max":1,"min":1,"queue_size":-1,"type":"fixed"},"management":{"keep_alive":"5m","max":5,"min":1,"queue_size":-1,"type":"scaling"},"refresh":{"keep_alive":"5m","max":1,"min":1,"queue_size":-1,"type":"scaling"},"rollup_indexing":{"max":4,"min":4,"queue_size":4,"type":"fixed"},"search":{"max":4,"min":4,"queue_size":1000,"type":"fixed_auto_queue_size"},"security-token-key":{"max":1,"min":1,"queue_size":1000,"type":"fixed"},"snapshot":{"keep_alive":"5m","max":1,"min":1,"queue_size":-1,"type":"scaling"},"warmer":{"keep_alive":"5m","max":1,"min":1,"queue_size":-1,"type":"scaling"},"watcher":{"max":1,"min":1,"queue_size":1000,"type":"fixed"},"write":{"max":2,"min":2,"queue_size":200,"type":"fixed"}},"total_indexing_buffer":105630924,"transport":{"bound_address":["10.77.10.83:9300"],"profiles":{},"publish_address":"10.77.10.83:9300"},"transport_address":"10.77.10.83:9300","version":"6.3.2"}